book/getting-started/linux-setup.md

```diff
@@ -169,17 +169,19 @@ cd ~/wallaroo-tutorial/wallaroo/machida
 make
 ```
 
-## Compiling Giles Sender, Receiver, and the Cluster Shutdown tool
+## Compiling Giles Sender, Receiver, Cluster Shutdown, and Cluster Shrinker tools
 
 Giles Sender is used to supply data to Wallaroo applications over TCP, and Giles Receiver is used as a fast TCP Sink that writes the messages it receives to a file, along with a timestmap. The two together are useful when developing and testing applications that use TCP Sources and a TCP Sink.
 
 The Cluster Shutdown tool is used to instruct the cluster to shutdown cleanly, clearing away any resilience and recovery files it may have created.
 
+The Cluster Shrinker tool is used to tell a running cluster to reduce the number of workers in the cluster and to query the cluster for information about how many workers are eligible for removal.
+
 To compile all three, run
 
 ```bash
 cd ~/wallaroo-tutorial/wallaroo/
-make build-giles-all build-utils-cluster_shutdown-all
+make build-utils-all
```

Comment:
Why is the instruction to build giles removed entirely? Do we not still use giles sender?